modules/adding-tls-termination.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/add-tls-termination.adoc
+
+:_content-type: PROCEDURE
+[id="nw-adding-tls-termination_{context}"]
+= Adding the tls termination on the AWS Load Balancer
+
+You can route the traffic for the domain `example.com` to pods of a `service` `example-service`, and add TLS termination on the AWS Load Balancer.
+
+.Prerequisites
+
+.Procedure
+
+. Deploy the operator and create an instance of the resource `AWSLoadBalancerController`:
++
+[source,yaml]
+----
+kind: AWSLoadBalancerController
+group: networking.olm.openshift.io/v1alpha1 <1>
+metadata:
+  name: cluster <2>
+spec:
+  subnetTagging: auto
+  ingressClass: tls-termination <3>
+----
+
+<1> Defines the API group of the `AWSLoadBalancerController` resource.
+<2> Defines the `aws-load-balancer-controller` instance.
+<3> The name of the `IngressClass` which the `aws-load-balancer-controller` will reconcile. This `IngressClass` will be created if it is not present. Additional `IngressClass`es can also be added which will be reconciled by the controller as long as their `spec.controller` is `ingress.k8s.aws/alb`.
```

Comment:
Additional  `IngressClass` values can also be added. They are reconciled by the controller when the `spec.controller` value is set to `ingress.k8s.aws/alb`.

Reply:
Ack'ed